docs/authentication_and_upload.md

```diff
@@ -56,8 +56,45 @@ authenticate with the server.
 
 ### prefix.dev
 
-To upload to [prefix.dev](https://prefix.dev), you need to have an account and a
-token. You can create a token in the settings of your account. The token is used
+#### OpenID Connect (OIDC)
```

Comment:
Can we start calling it something like Trusted publishing via OIDC or so?

OIDC is very technical and trusted publishing is maybe easier to grasp (also if you know pypi of course).